src/GHC/Util.hs

```diff
@@ -13,11 +13,13 @@ module GHC.Util (
   , Located
   , readExtension
   , commentText, isCommentMultiline
-  , declName
+  , nameOfDecl, nameOfModule
```

Comment:
These seem very OCaml-style names for the things. I think declName and moduleName would be the more Haskelly names.

Reply:
`moduleName` has a clash with HSE, so I went with `nameOfModule` and if we are doing `nameOfModule` then `nameOfDecl` is changed for consistency.

Reply:
Can we do `modName` or `ghcModuleName`? (And then remove the `ghc` prefix in a future state)

Reply:
Yes, we can do that.

Reply:
This is done.